Header header
float64 distance
int64 numTurns
